projects/kit/components/input-tag/input-tag.style.less

```diff
@@ -79,15 +79,7 @@
 }
 
 .content {
-    padding: 0;
     align-items: flex-start;
-
-    :host[data-tui-host-size='s'] &,
-    :host[data-tui-host-size='m'] & {
-        padding: 0 0 0 0.75rem;
-        padding-inline-start: 0.75rem;
-        padding-inline-end: 0;
-    }
```

Comment:
All paddings are already set via [.textfield-content()](https://github.com/Tinkoff/taiga-ui/blob/main/projects/core/styles/mixins/textfield.less#L238-L240) mixin.

**Why it works before ?**
It was balanced by right-padding [here](https://github.com/Tinkoff/taiga-ui/pull/983/files#diff-facdbe6225bfb5c51ae780cf893edf3df1789303980c9da156a92e766f32119eL89).
But it was removed and it produces bug.